src/views/CrosswordpuzzleView.vue

```diff
@@ -50,15 +137,41 @@ function evaluateSolution() {
   if (isCorrect) {
     evaluationModalContext.value.title = "Congratulations! 🥳";
     evaluationModalContext.value.text = "Everything right!";
+    questions.forEach((question) => {
+      answers.add({
+        answer: question.answer,
+        correctAnswer: question.answer,
+        question: question.questionText,
+        correct: true,
+      });
+    });
   } else {
+    questions.forEach((question, index) => {
+      if (!wrongQuestions.has(index + 1)) {
+        answers.add({
+          answer: question.answer,
+          correctAnswer: question.answer,
+          question: question.questionText,
+          correct: true,
+        });
+      }
+    });
     evaluationModalContext.value.title = "Not the correct answers";
     evaluationModalContext.value.text = "Maybe the next time";
   }
+
   const gameResult: GameResult = {
     correctTiles: numberOfTiles - wrongTiles,
     numberOfTiles: numberOfTiles,
     configuration: configuration,
+    answers: [],
+    duration: Date.now() - time,
```

Comment:
This calculates the duration in milliseconds, right?
Aren't we using seconds in other games?